### ISSUE TITLE
spec/oauth TOC Links Unresolvable

### ISSUE THREAD

**Issue by dezren39** [mastodon/documentation#1139]
https://docs.joinmastodon.org/spec/oauth/

In the TOC At the top,
The 3 Methods Listed don't link correctly.
I think this is because the chapter links are actually into the methods pages and whatever automation works on these links is confused by that. But I could be entirely wrong.
The other links in the TOC work OK.
The actual links if you scroll down in the page work fine.

```
[What is OAuth?](https://docs.joinmastodon.org/spec/oauth/#intro)
[OAuth 2 endpoints implemented](https://docs.joinmastodon.org/spec/oauth/#implementation)
[GET /oauth/authorize](https://docs.joinmastodon.org/spec/oauth/HAHAHUGOSHORTCODE-s4-HBHB)
[POST /oauth/token](https://docs.joinmastodon.org/spec/oauth/HAHAHUGOSHORTCODE-s5-HBHB)
[POST /oauth/revoke](https://docs.joinmastodon.org/spec/oauth/HAHAHUGOSHORTCODE-s6-HBHB)
[Common gotchas](https://docs.joinmastodon.org/spec/oauth/#gotchas)
```

Chrome v111

**Comment by trwnh** [comment]
oh, it's because the headings are linked to a different section -- the `{{<` should be `{{%` to make them non-rendering shortcodes. Or perhaps it would be best to just reformat them so that the links are below the headers rather than inside them...

**Comment by trwnh** [comment]
Should be fixed in #1110 now
